mlrun/api/api/endpoints/secrets.py

```diff
@@ -155,11 +154,8 @@ async def list_project_secrets(
 def add_user_secrets(
     secrets: schemas.UserSecretCreationRequest,
 ):
-    if secrets.provider != schemas.SecretProviderName.vault:
-        return fastapi.Response(
-            status_code=HTTPStatus.BAD_REQUEST.vault,
-            content=f"Invalid secrets provider {secrets.provider}",
-        )
-
-    add_vault_user_secrets(secrets.user, secrets.secrets)
-    return fastapi.Response(status_code=HTTPStatus.CREATED.value)
+    # vault is not used
```

Comment:
maybe add a todo to remove this endpoint

Reply:
@theSaarco wdyt?

Reply:
I'm fine with deprecating the API, but the endpoint is just part of the flow, and I do not want to completely remove the Vault logic from our code as we might need to revive it at some point.

Reply:
so no reason to actually deprecate it if we want to leave the code, I'll leave it like this